app/models/expression_file_info.rb

```diff
@@ -6,7 +6,7 @@ class ExpressionFileInfo
   field :library_preparation_protocol, type: String
   field :units, type: String
   field :biosample_input_type, type: String
-  field :multimodality, type: String
+  field :modality, type: String
```

Comment:
It's OK to wait.  This can be handled by a migration.  I tried setting it to `Array` here and elsewhere and encountered issues, so it seems reasonable to bundle the data type refinement with the UI refinement for later.